public/Sjofn/homework1.html

```diff
@@ -0,0 +1,72 @@
+<html lang="en">
+<head>
+    <title>Homework 1</title>
+    <style>
+        div {
+            background-color: dodgerblue;
+        }
+        #myId{
+            color: tomato;
+            background-color: cadetblue;
+            size: A3;
+        }
+        .myClass{
```

Comment:
Please, provide accurate and clear names to IDs, classes, etc.
'_section1_' class name would be better in this case.

Reply:
You may also provide a class name or an ID to each section, - it would make the HTML more clear to a reader.